content/blog/eng/v-protetto17-5-2024.md

```diff
@@ -0,0 +1,120 @@
+
+![Please wear your favorite hacker hoodie before trying the workshop]( /static/images/vp-may-2024/hack-the-box-v0-a56fw7h8a2aa1.webp "Hack the Box Wallpaper")
+*Please wear your favorite hacker hoodie before trying the workshop*
+
+
+### Preparing the workshop
+
+
+![Topics of the workshop](/static/images/vp-may-2024/topics.png "Topics of the workshop")
+*Topics we focused on*
+
+Let's not hide ourself behind a finger: Hacking an application is not a trivial task. It requires a fond knowledge of how application works, several tools are used to diagnose and to try to break the application. But preparing all this stuff requires quite a lot of time.
+
+So we prepared a virtual machine with all the needed tools. We try to create a lightweight yet simple to prepare VM, so we started from a [Debian 11 VM](https://mac.getutm.app/gallery/debian-11-ldxe), in which we installed:  
+
+- the challenge itself
+- a decoder and repeater, we used [Burp Suite community edition](https://portswigger.net/burp/communitydownload)
+- a simple proxy - we used [FoxyProxy](https://getfoxyproxy.org/)
+- a reverse proxy - we used [ngrok](https://ngrok.com/) 
+
+### On to the job!
+
+![Vulnerability discovered! Can you guess how?]( /static/images/vp-may-2024/vulnerability.png "JerryTok vulnerability disclosed") 
+*Vulnerability discovered! Can you guess how?*
+
+Workshop itself was subdivided in:
+
+- a theoretical introduction to SSTI
+- hands on the machine
+
+During the introduction, we discovered what is an SSTI, several tool to analyze if a page can be affected by this problem and how to mitigate it.
+Hands on was guided by Alessio, and some volunteers shared their screen in order to have feedback directly from the learners. The machine used some basic security system, so hacking is not trivial.
+
+Overall, it was a though yet interesting experience.
+
+# How to develop cognitive flexibility?
+![]()
+In this webinar you will discover ...
+
+* ... some of the most common mistakes you can make when evaluating situations
+* ... that NOT to follow your intuition can be highly convenient
+* ... techniques that will help you develop cognitive flexibility, improving the efficacy by which you deal with complex situations
+* ... books and movies that may be useful in your training journey.
+
+
+PS: Arduino Mancini, the business coach who will hold the webinar and well-known to many in Facile.it, recommended that during the webinar you have 6 toothpicks at hand 😲
```

Comment:
who **held** the webinar

Reply:
Io qui scriverei: Arduino Mancini, the business coach who held the webinar, **well known** to many at Facile, recommended having 6 toothpicks at hand during the webinar.